ports/osqp-eigen/vcpkg.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "osqp-eigen",
+  "version": "0.7.0",
+  "port-version": 2,
+  "description": "Simple Eigen-C++ wrapper for OSQP library",
```

```suggestion
  "description": "Simple C++ wrapper for the osqp library",
```
https://robotology.github.io/osqp-eigen/